pages/community/precice-workshop.md

```diff
@@ -0,0 +1,101 @@
+We typically help new users design their coupling projects, discuss feature requests, or resolve bugs together.
+
+<img class="img-responsive center-block" src="images/events/precice2020-support.jpeg" alt="User support session" style="width: 70%; margin:auto;">
+
+## Giving back to the community
+
+The workshop is open to everyone, but we are aware that not everyone is able to attend,
+while we want to be able to refer to the content ourselves.
+For this reason, we release talk recordings on [YouTube](https://www.youtube.com/c/preCICECoupling), gradually after each workshop.
+
+We also publish [training material developed together with the community](https://github.com/precice/community-training).
+Note, however, that this cannot capture the hands-on collaborative experience that the workshop offers.
+
+## Small investment, large benefit
+
+We structure the workshop schedule in a way that you can attend selected parts of it,
+especially if you are already an experienced user, or if you just want to see
+what the latest updates and new applications are.
+
+We also set the participation fee just enough to cover the expenses,
+offering discounts for academia and supporting the participation
+of students and international researchers that would otherwise not be able to join.
+
+In return, the whole preCICE team works together to make sure that we all
+have the best possible experience and learning.
+
+Don't just take our word for it, see what previous participants wrote:
+
+> The task by task design of the course was nice to deeper understand how preCICE works.
+> Talk with the developers about our applications and problems was very useful.
+> **I come back to my lab with a lot of new ideas and solutions.**
+> I would come again to follow the updates of preCICE,
+> to find some solutions to the problem that I face and hopefully present my
+> application coupling if the results and conclusions are interesting.
+
+> Really enjoyed all the **networking opportunities.**
+
+> I really like the **interaction with the other users** in the preCICE community.
+> **It's a good place to get ideas and talk about your work.**
+> I would come again to get to know the updates and get some hands-on user help.
+
+## History
+
+So far, we have aimed for annual workshops rotating between Munich and Stuttgart, the historical homes of preCICE.
+In 2025, we are organizing our first workshop together with the community.
+Let us know if you would like to host and help us organize one of the next workshops,
+in particular if this makes sense in the context of a research funding proposal and/or an active local community.
+
+| N  | Date                                      | Place       | Host                                 |
+|--- | ---                                       | ---         | ---                                  |
+| 6  | [September 2025](./precice-workshop-2025) | Hamburg     | Helmut-Schmidt University of Hamburg |
```

Comment:
This page does not yet exist. Would be good to have stub with logo already.

Reply:
I created a page, but I cannot currently render the website locally to test it.
No clue why, I could build and render it recently.